### DISCUSSION TITLE
Subscribing to OPCUA nodes with a constant value

### DISCUSSION THREAD

**Discussion by KishanKishore** [FreeOpcUa/opcua-asyncio#584] [Q&A] [answered]
When subscribing to a node with a constant value using ```asynua```, there are no updates from the OPCUA server and the ```datachange_notification``` method is not repeatedly executed. 
- Is this expected behavior ? 
- Is it possible to make some changes on the client side so that the OPCUA server keeps sending notifications for such nodes with the same value?

**Comment by AndreasHeine** [comment, marked as answer]
https://reference.opcfoundation.org/v104/Core/docs/Part4/5.13.1/

why should data been send if it has not changed since last notification?

**Reply by KishanKishore** [reply]
@AndreasHeine Thanks for the quick response as always. I was actually going through the same page. Just need to confirm that this is expected behavior.

**Reply by AndreasHeine** [reply]
thats by the way the cool thing on subscriptions you get only datachanges ;)

**Reply by AndreasHeine** [reply]
if we would change it, it would be just cyclic polling.... a subscription is more like longpolling!

**Reply by KishanKishore** [reply]
It is confirmed here: https://reference.opcfoundation.org/v104/Core/docs/Part4/5.12.1/
<img width="1062" alt="Screenshot 2021-06-02 at 9 03 36 PM" src="https://user-images.githubusercontent.com/59863682/120506934-1957a200-c3e6-11eb-8582-e0f3399eb1ea.png">

Thanks for the help!
